src/main/java/org/opentripplanner/model/Trip.java

```diff
@@ -15,6 +15,10 @@
 
     private String tripShortName;
 
+    private String tripPublicCode;
+
+    private String tripPrivateCode;
+
```

Comment:
I suggest mapping the Netex `tripPublicCode` into `tripShortName` - maybe not exactly the same - but close enogh. Is there something in Netex that mappes better to GTFS `tripShortName`?

We should discuss on tuesday what to do with `tripPrivateCode`. The options are:
 1. Add it to trip (as done here) - and add documentation.
 2. Add a new field `Trip.extNetex` with type `NetexTripExtention`. The `NetexTripExtention` will have one field and that is `tripPrivateCode`. 
 3. Mape and index(Map) from Trip.id to 'tripPrivetCode' and store it in GraphIndex.

Considerations:
- Is this a "Entur" specific hack?
- Is it likely that we need to add more fields to Trip later, then option 2 is best. Also this make a good example on how to add stuff like this.